.github/compatibility-test-matrices/release-v2.5.x/transfer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "chain-a": ["release-v2.5.x", "v2.4.2"],
+  "chain-b": ["release-v2.5.x", "v2.4.2"],
```

Comment:
All compatibility tests should test against all supported versions. If we were only doing a v2.5 release (and not v3.4, 4.2 etc), this would not test against the other releases (and we likely would run compatibility tests for the other releases)